docs/data.md

```diff
@@ -205,7 +243,7 @@ You can also call data from the xml definition of a `data` tag directly:
 
 Attributes|Type|Use|Description
 ---|---|---|---
-`name`|string|optional|Name of the `<entity>`.
+`name`|string|optional|Name of the `<entity>`. Camel case is used for the entity name.
```

Comment:
"Use camel case for entity names."

Reply:
@dobooth Updated the changes. Please review it.